src/plugins/exampleUser/plugin.js

```diff
@@ -0,0 +1,29 @@
+/*****************************************************************************
```

Comment:
I think we should move this to `/openmct/example` folder

Reply:
Good catch, I agree. It's not ever intended for production usage.